modules/banner/react/lib/Banner.tsx

```diff
@@ -67,7 +67,7 @@ const BannerWrapper = styled('button')<BannerProps>(
   })
 );
 
-const BannerIcon = styled(SystemIcon)({
+const BannerIcon = styled(SystemIcon)<BannerProps>({
```

Comment:
What was the reason for adding `<BannerProps>` here?

Reply:
It was needed before so styled had access to the direction (when we passed it manually) but not anymore. I'm removing this.